script.py

```diff
@@ -0,0 +1,72 @@
+import subprocess
+import sys
+import boto3
+
+def mongodump(database,collections,outpaths):
+     for i in range(len(collections)):
+          command = ["mongodump", "--uri", database, "--collection", collections[i],"--out",outpaths[i]]
```

Comment:
if you are going to dump the db completely, no need to use `--collection`

Reply:
no i am dumping it collection by collection